packages/client-search/src/methods/index/getObject.ts

```diff
@@ -2,13 +2,13 @@ import { encode } from '@algolia/client-common';
 import { MethodEnum } from '@algolia/requester-common';
 import { RequestOptions } from '@algolia/transporter';
 
-import { GetObjectOptions, ObjectWithObjectID, SearchIndex } from '../..';
+import { GetObjectOptions, Hit, SearchIndex } from '../..';
 
 export const getObject = (base: SearchIndex) => {
   return <TObject>(
     objectID: string,
     requestOptions?: RequestOptions & GetObjectOptions
-  ): Readonly<Promise<TObject & ObjectWithObjectID>> => {
+  ): Readonly<Promise<Hit<TObject>>> => {
```

Comment:
there's no highlighting on getObject I think

Reply:
That's right indeed. I'm just gonna change the `*Response` types.

Reply:
See fc268e0eee87b835fe85eeaf0d9225c31e9d5b35.